Week2/5.js

```diff
@@ -0,0 +1,17 @@
+let z = 7.25;
+
+console.log(z);
+
+let a = Math.round(z);
+
+console.log(a);
+
+// to compare and see the values :
+
+console.log(a > z);
```

Comment:
The assignment is as follows:

```
5.5 So now we have z and a find a way to compare the two values and store the highest of the two in a new variable.
5.6 console.log the highest value.
```

So can you store the highest value in a variable and then console.log that one?

Reply:
yes, it is explicit after doing console.log(a > z) that z has bigger value than a. and z is assigned to `highestNumber` in order to store highest value in a new variable. do I get it wrong Marciano?

Reply:
I just figured out that `Math.max()`  can be used to store the highest value of variables. so, `let highestNumber = Math.max(z ,a)` would give us what is wanted.

Reply:
Ah, yeah your code is techically correct. But you are deciding for yourself which value is higher.
You do this

```
console.log(a > z);
```

But, you don't store the `highest number` in a variable.

And after that you do this 
`let highestNumber = z;`

But how does your code know that `z` is higher than `a`? Do you understand what I mean?

And yeah, if you do this:
`let highestNumber = Math.max(z ,a)` then the code knows what the highest value is, because it is stored in `highestNumber`. So the differnce is that now the code decides that the value is higher, but before, you decided which value is higher.

If it's not clear I will try to explain it better:)